src/Markbench/Driver/ParsedownExtraDriver.php

```diff
@@ -44,7 +44,7 @@ public function getName()
      */
     public function getDialect()
     {
-        return 'extra';
+        return 'gfm & extra';
```

Comment:
isn't extra a superset of gfm? I mean if you look at it in very detail there are probably things that do not match, but both have no detailed spec so I'd leave `extra` here.

Reply:
Oh well, a lot has changed since I looked at these in detail :)

Reply:
GFM used to be as about underspecified as Gruber Markdown, so you're not wrong to think it didn't have a detailed spec at one point :)

GitHub ended up writing it more formally as a CommonMark extension at some point, but kept the same name (I remember a couple years ago the GitHub parser was horrible and kept breaking my comments when they changed things slightly! :p).

Reply:
btw, I have just seen https://github.com/Parsemd/Parsemd-PHP is there something interesting coming out of that or was it just an experiment?

Reply:
cool, thanks for the detailed response :)

Reply:
No worries :)

Something I ended up being quite pleased with is the abstract emphasis implementation passed all the CommonMark emphasis examples (and I'm pretty sure also agrees completely with spec) when specialised to the CommonMark specifics (marker run length, markers etc..)